.gitignore

```diff
@@ -11,3 +11,4 @@ node_modules
 output
 
 example/example.js
+github_deploy_key
```

Comment:
Would you mind rebasing this PR so that changes unrelated to your work are not included?

Reply:
this came from current master https://github.com/srghma/purescript-halogen/commit/d2f59b990a51152ca6cd53dee4dcd295f85d5a2d

you see it in this pr becuase I'm trying to merge to next-6 branch, that is not rebased on current master

I have merged halogen/master to my branch

this should disappear once halogen/next-6 branch is rebased on halogen/master